src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -0,0 +1,217 @@
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Output,
+  ViewChild
+} from '@angular/core';
+
+import {
+  SkyFileItem
+} from './file-item';
+
+import {
+  SkyFileAttachmentChange
+} from './file-attachment-change';
+
+import {
+  SkyFileAttachmentService
+} from './file-attachment.service';
+
+import {
+  SkyFileItemService
+} from './file-item.service';
+
+@Component({
+  selector: 'sky-file-attachment',
+  templateUrl: './file-attachment.component.html',
+  styleUrls: ['./file-attachment.component.scss'],
+  providers: [
+    SkyFileAttachmentService,
+    SkyFileItemService
+  ]
+})
+export class SkyFileAttachmentComponent {
+  @Output()
+  public fileChanged = new EventEmitter<SkyFileAttachmentChange>();
```

Comment:
Should be `fileChange` since this event doesn't auto-complete after emitting (past tense events auto-complete after emit, see: https://docs.blackbaud.com/skyux-team-docs/style#eventsoutputs).

Reply:
This event should also be completed in the `ngOnDestroy` lifecycle hook.